packages/fx-core/tests/component/driver/botAadApp/create.test.ts

```diff
@@ -184,4 +184,31 @@ describe("botAadAppCreate", async () => {
       }
     );
   });
+
+  it("should be good when reusing existing bot in env", async () => {
+    const originalValues = {
+      BOT_ID: process.env.BOT_ID,
+      SECRET_BOT_PASSWORD: process.env.SECRET_BOT_PASSWORD,
+    };
+    // mock env
+    process.env.BOT_ID = expectedClientId;
```

Comment:
It's better to use mockedEnv

Reply:
Good suggestion! It's the first time for me to use `mockedEnv`.